data/quickstarts/oneapi-aikit-notebook.yaml

```diff
@@ -87,7 +87,7 @@ spec:
       review:
         instructions: |-
           #### Verify that the sample notebook ran successfully
-          Did all cells of the notebook run successfully and printed out the statement "[CODE_SAMPLE_COMPLETED_SUCCESFULLY]" at the end?
+          Did all cells of the notebook run successfully and printed out the statement "[CODE_SAMPLE{_}_{_}COMPLETED_SUCCESFULLY]" at the end?
```

Comment:
could you remove `{_}_{_}` and add the escape character before every underscore?
`CODE\_SAMPLE\_COMPLETED\_SUCCESFULLY`

Reply:
Done